documentation/latest/gsg/running_workloads.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: documentation
+title: Run workloads
+---
+
+Now that the devices are running, we need to run some container workloads on
+these devices, for that, the first thing that we need to do, it's to make sure
+that devices are online:
+
+```shell
+---> kubectl get edgedevices
+NAME        AGE
+camera-ny   33m
+--->
+```
+
+To run a workload, we need to create a EdgeWorkload config, that it's looks like
+this:
+
+```yaml
+apiVersion: management.project-flotta.io/v1alpha1
+kind: EdgeWorkload
+metadata:
+  name: workload
+spec:
+  deviceSelector:
+    matchLabels:
+      app: camera
+  type: pod
+  pod:
+    spec:
+      containers:
+        - name: workload
+          image: quay.io/project-flotta/nginx:1.21.6
+```
+
+
+This workload will look for devices that has the label app=camera, so we need to
+tag our device with that tag:
+
+```shell
+kubectl label edgedevice camera-ny app=camera
```

Comment:
Add a note stating that "camera-ny" is the name of the targeted edgedevice?

Reply:
added a comment just before ;-)